index.md

```diff
@@ -52,18 +53,25 @@ If you’re still in need of clarification, there are [a number](https://us.pyco
 All LRUG participants are accountable for their own behaviour. If you’ve behaved badly elsewhere, that may count against you here, because of the effect it has on other attendees.
 
 If you experience or witness any harassing behaviour, please report it
-to a member of the [volunteer team](#volunteer-team).
+to a member of the [organisation team](#organisation-team).
 
-#### Volunteer team
+## Organisation team
 
- * [Aanand](http://aanandprasad.com/) ([@aanand](http://twitter.com/aanand), [aanand.prasad@gmail.com](mailto:aanand.prasad@gmail.com))
- * [Tom](http://codon.com/) ([@tomstuart](http://twitter.com/tomstuart), [tom@codon.com](mailto:tom@codon.com))
+LRUG is run by a team of volunteers from the London Ruby community:
 
-If you'd like to join the volunteer team please contact
-[Murray Steele](http://h-lame.com/)
-([@hlame](http://twitter.com/hlame),
-[murray.steele@lrug.org](mailto:murray.steele@lrug.org)) or one of the
-other [volunteers](#volunteer-team).
+ * [James Adam](http://lazyatom.com) ([@lazyatom](https://twitter.com/lazyatom), [james.adam@lrug.org](mailto:james.adam@lrug.org))
+ * [Chris Lowis](http://blog.chrislowis.co.uk/) ([@chrislowis](https://twitter.com/chrislowis), [chris.lowis@lrug.org](mailto:chris.lowis@lrug.org))
+ * [Melinda Seckington](http://missgeeky.com/) ([@mseckington](https://twitter.com/mseckington), [melinda.seckington@lrug.org](mailto:melinda.seckington@lrug.org))
+ * [Tatiana Stantonian](https://github.com/binaryberry) ([@tomstuart](https://twitter.com/binaryberry), [tatiana.stantonian@lrug.org](mailto:tatiana.stantonian@lrug.org))
```

Comment:
One of these days I won't need the incredible @tomstuart typo-detection bot to fix things for me...  fixed and rebased, thanks!